examples/charging-server-simulator/src/main/resources/dictionary.xml

```diff
@@ -7616,4 +7615,199 @@
 
   <!-- ************************ END 3GPP S13/S13' AVPS ******************* -->
 
+  <!-- ***************************** 3GPP Sy AVPS ************************ -->

```

Comment:
This should be modified in all the instances of  the `dictionary.xml` file, I understand it's not worth to propagate while not finished, but we should not forget about it. Main file is https://github.com/RestComm/jdiameter/blob/master/core/mux/common/config/dictionary.xml